.github/build-mandrel-images.sh

```diff
@@ -18,10 +18,13 @@ do
 done
 
 # Get all the final versions, reverse sort them, and keep only the first for each major.minor (e.g. 20.1)
-LATEST_FINAL_VERSIONS=$(sed 's/ /\n/g' <<< ${VERSIONS[@]} | grep Final | sort -r | sort -k1,1 -k2,2 -k5,5 -t'.' --unique)
+LATEST_FINAL_VERSIONS=$(tr ' ' '\n'  <<< "${VERSIONS[@]}" | grep Final | sort -r | sort -k1,1 -k2,2 -k5,5 -t'.' --unique)
```

Comment:
Following https://github.com/quarkusio/quarkus-images/pull/93#discussion_r472976197 please edit this as well to be:
```suggestion
LATEST_FINAL_VERSIONS=$(tr ' ' '\n'  <<< "${VERSIONS[@]}" | grep Final | grep java11 | sort -r | sort -k1,1 -k2,2 -t'.' --unique)
```

Reply:
Done. 

Let's see what the CI has to say.